app/clients/file_client.rb

```diff
@@ -13,7 +13,7 @@ class FileClient
   # - the contents of the file.
 
   def self.save(user_id, folder, filename, content)
-    filepath = self.filepath user_id, folder, filename
+    filepath = filepath user_id, folder, filename
```

Comment:
I know that in this case using ```self``` is not required, and some style guides discourage it.

However I think that in some cases it makes the code more readable. In this case, where the method and the variable storing the result have the same name, this is specially true.

I'm going to revert this particular change.